src/public/modules/forms/admin/controllers/view-responses.client.controller.js

```diff
@@ -61,11 +63,12 @@ function ViewResponsesController(
 
   // Trigger for export CSV
   vm.exportCsv = function () {
+    const userDetails = JSON.parse($window.localStorage.getItem("user"))
     let params = {
       formId: vm.myform._id,
       formTitle: vm.myform.title,
+      userEmail: userDetails.email,
```

Comment:
@shuli-ogp I'm still not sure this is a good solution because this will still result in the `userEmail` unintentionally being sent as a query parameter when the user downloads encrypted responses. Do you think it might be better to extract the user information from `$window` in the GA service itself?

Reply:
Thanks; please see #192 which fixes this